modules/ROOT/pages/monitoring-dashboards.adoc

```diff
@@ -88,11 +88,37 @@ You can view all graphs for different time ranges by selecting the desired time
 image::monitoring-dashboards-time-ranges.png[]
 * Classic application monitoring dashboard:
 +
+--
 image::monitoring-dashboards-cbd92.png[]
 
+Depending on the selected time range, the graphs show the average, minimum, and maximum:
```

Comment:
Is it redundant to have the intro sentence since the table header says the same thing?